assemblies/assembly-audit-log.adoc

```diff
@@ -34,6 +30,14 @@ Audit logs are not forwarded to the internal log store by default because this d
 [role="_additional-resources"]
 .Additional resources
 
-* For more information, see link:https://docs.openshift.com/container-platform/latest/observability/logging/cluster-logging.html[{ocp-brand-name}  - About Logging]
+* For more information about logging in {ocp-short}, see link:https://docs.openshift.com/container-platform/latest/observability/logging/cluster-logging.html[About Logging]
+* For a complete list of fields that a {product-short} audit log can include, see xref:ref-audit-log-fields.adoc_{context}[]
```

Comment:
What is the full file path for `ref-audit-log-fields.adoc`? When I click on it from the preview, it returns a 404.

Reply:
I've noticed that most internal links 404 in GitHub previews, including links that work in prod. I've asked why that is / if it's expected behavior, but I don't recall getting a clear answer.

Reply:
FWIW, I didn't change the link, just the link text. And the link does appear to work in prod. 
https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/getting_started_with_red_hat_developer_hub/assembly-audit-log#assembly-audit-log

Reply:
Is `xref:ref-audit-log-fields.adoc` pointing to a file in Red Hat Developer Hub documentation, or in OpenShift Container Platform documentation?

Reply:
Might be missing [<link text>] so ... something like ...

xref:ref-audit-log-fields.adoc_{context}[Audit log fields]

See https://redhat-documentation.github.io/supplementary-style-guide/#cross-references

Reply:
In these docs, xrefs are typically formatted as xref:filename_{context}[] and it appears that when [] is used, it automatically fills in the title of the xref when rendered. I didn't do xrefs this way before joining this team, and I notice that the xrefs are rendering differently locally than in GitHub preview, so I did ask the team in another comment if that was expected behavior, but I haven't gotten a response yet.

Reply:
Resolving as it seems to be a Pantheon-specific thing.